clang-tools-extra/test/clang-tidy/checkers/readability/redundant-inline-specifier.cpp

```diff
@@ -0,0 +1,141 @@
+// RUN: %check_clang_tidy %s readability-redundant-inline-specifier -std=c++17 %t
+
+template <typename T> inline T f()
```

Comment:
according to C++ insights this inline is not redundant, please verify.
Wont inline keyword force change here of linkage to internal ?
Maybe this should also be checked.

Simply because if i call this function from one file, and then i use forward declaration in other file, then without inline keyword code may still compile and symbol will be emitted.